spec/models/question_spec.rb

```diff
@@ -0,0 +1,58 @@
+# == Schema Information
+#
+# Table name: questions
+#
+#  id           :bigint           not null, primary key
+#  for_banks    :boolean
+#  for_partners :boolean
+#  title        :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+require "rails_helper"
+
+RSpec.describe Question, type: :model do
```

Comment:
Let's add specs for the validations you've added. One of my personal favorites for that is https://github.com/thoughtbot/shoulda-matchers which makes it fairly easy to just copy and paste to get test coverage :)

Reply:
Thanks for the suggestion. I'll look into using this gem.

Reply:
I used this gem to test some validations in the model spec. Thanks for introducing me to it.